python/pyspark/sql/session.py

```diff
@@ -151,7 +151,7 @@ def getOrCreate(self):
             new one based on the options set in this builder.
 
             This method first checks whether there is a valid global default SparkSession, and if
-            yes, return that one. If no valid global default SparkSession exists, the method
+            there is, return that one. If no valid global default SparkSession exists, the method
```

Comment:
If you change this from `yes` to `there is`, it breaks the remaining `If no`.
I'd like to recommend not to do this in this PR.

Reply:
How about just "If not," to avoid repeating the predicate.